.github/workflows/ci.yml

```diff
@@ -31,25 +47,27 @@ jobs:
       - name: Generate documentation
         run: npx jsdoc -c jsdoc.json
 
+  # Deploy GitHub pages
   build-and-deploy:
     needs: lint-and-test
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

Comment:
https://github.com/orgs/community/discussions/37267 - if people want to learn more about this line